### PR TITLE
Fixed some bugs in go-bindata-assetfs utility.

### PR DESCRIPTION
**Pull request by khaos-one**
Utility now should be Winodws-compatible.

### CODE REVIEW

File: go-bindata-assetfs/main.go [hunk 1]
```diff
@@ -47,21 +45,30 @@ func main() {
 			done = true
 		}
 	}
+	/* Note by khaos: Prefix omitted due to strange behaviour. */
 	fmt.Fprintln(out, `
 func assetFS() *assetfs.AssetFS {
-	for k := range _bintree.Children {
```
Comment (elazarl) [inline go-bindata-assetfs/main.go]: Can you explain the reason for this change?

**Comment by elazarl** [comment]
Hey, thanks for the PR. Looks reasonable except the part with the children.

Care to explain why did you remove the prefix?

**Comment by khaos-one** [comment]
Yeah, sorry, didn't think this dirty hack will get to the final commit.
The purpose of the hack is that FS driver does not correctly work with this prefix and it's removal fixed the issue.

For example, if I have two files "index.html", and "app.html", the overall assetFS.Prefix becomes "app.html" so when one is trying to access "index.html", the system reports that it cannot find "app.html/index.html". This issue of course should be fixed in the inner software layer.

I can reroll the PR without this hack.

File: go-bindata-assetfs/main.go [hunk 2]
```diff
@@ -47,21 +45,30 @@ func main() {
...
 }`)
+	// Close files BEFORE remove calls (don't use defer).
+	in.Close()
+	out.Close()
 	if err := os.Remove("bindata.go"); err != nil {
 		fmt.Fprintln(os.Stderr, "Cannot remove bindata_assetfs.go", err)
 	}
 }
 
 func isInPath(filename string) bool {
```
Comment (shurcooL) [inline go-bindata-assetfs/main.go]: This entire `isInPath` func looks suspiciously close to [`exec.LookPath`](http://godoc.org/os/exec#LookPath), any reason not to use that?

Comment (shurcooL) [inline go-bindata-assetfs/main.go]: Maybe not necessarily in this PR, but in general.

Comment (khaos-one) [inline go-bindata-assetfs/main.go]: I agree.

**Comment by khaos-one** [comment]
So, should I reroll the PR, or just push new commit without the hack?

**Comment by elazarl** [comment]
I think that resubmitting is the best.

Dmitry, thanks. That would be a better idea.

**Pull request closed without merging**